### PR TITLE
fix: update pylint to ignore import-errors

### LINKED ISSUE #5926: Regression of pylint config - import-error's are no longer disabled
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Super-linter version available?

- [X] I am using the latest Super-linter version.
- [X] I can reproduce the issue running Super-linter using complete version identifier (example: vX.Y.Z), and not just with a partial one (example: vX)

- [X] I am using the super-linter/super-linter action or container image, and not the deprecated github/super-linter action or container image.

### Are you resonably sure that it's a Super-linter issue, and not an issue related to a tool that Super-linter runs?

- [X] I think that this is a Super-linter issue.

### Current Behavior

When using the latest version of super linter import errors are not ignored!

This means i will have to update each repos config to ignore import errors despite previously having no expectation for import errors to succeed while using super-linter

This seems to have been changed as of: https://github.com/super-linter/super-linter/pull/5252

### Expected Behavior

Import errors arnt reported by pylint inside of the super-linter i.e. the following error shouldnt show up:

E0401: Unable to import '***' (import-error)

### Super-Linter version

```markdown
super-linter/super-linter/slim@3fe03abab2eafb293ace16d4a3b07aeabcb3f1a0

(v6.7.0)
```

### Super-linter configuration

```yaml
- name: checkout code
  uses: actions/checkout@a5ac7e51....
  with:
    fetch-depth: 0
- name: Lint
  uses: super-linter/super-linter/slim@3fe03abab2eafb293ace16d4a3b07aeabcb3f1a0
  env:
    DEFAULT_BRANCH: main
    GITHUB_TOKEN: {token}
    #disable non pylint things e.g. VALIDATE_PYTHON_MYPY: false
    LINTER_RULES_PATH: .github/linters #fails without this aswell
```

### Relevant log output

```shell
E0401: Unable to import '***' (import-error)
```

### Steps To Reproduce

Create a python file with 3rd party library imports (e.g. terrasnek)

Run the super linter using github actions!

I am having trouble running the docker file myself to create a simple step by step guide to reproduce this (being on arm64 probably doesnt help) however you can almost certainly follow the steps from previous issue: https://github.com/super-linter/super-linter/issues/3180 but with update docker commands

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Shop-kins**
Adds back import-error to the pylint disable rules that was removed as of: https://github.com/super-linter/super-linter/pull/5252

Fix #5926

# Proposed changes

update .python-lint to include `disable=import-error`

Also adds a new test case for import errors

## Readiness checklist

In order to have this pull request merged, complete the following tasks.

### Pull request author tasks

- [ ] I checked that all workflows return a success.
- [x] I included all the needed documentation for this change.
- [X] I provided the necessary tests.
- [X] I squashed all the commits into a single commit.
- [X] I followed the [Conventional Commit v1.0.0 spec](https://www.conventionalcommits.org/en/v1.0.0/).
- [N/A] I wrote the necessary upgrade instructions in the [upgrade guide](../docs/upgrade-guide.md).
- [X] If this pull request is about and existing issue,
- [X] I added the `Fix #ISSUE_NUMBER` or `Close #ISSUE_NUMBER` text to the description of the pull request.

### Super-linter maintainer tasks

- [x] Label as `breaking` if this change breaks compatibility with the previous released version.
- [x] Label as either: `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`.
- [x] Add the pull request to a milestone, eventually creating one, that matches with the version that release-please proposes.

### CODE REVIEW

File: TEMPLATES/.python-lint [hunk 1]
```diff
@@ -2,3 +2,10 @@
 
 # Use multiple processes to speed up Pylint.
 jobs=0
+
+[MESSAGES CONTROL]
+
+# Disable the message, report, category or checker with the given id(s). You
+# can either give multiple identifiers separated by comma (,) or put this
+# option multiple times (only on the command line
+disable=import-error
```
Comment (ferrarimarco) [inline TEMPLATES/.python-lint:11]: Can you add a comment to explain why we need to disable this check?

Also, can you remove the explanation about how to use the `disable` directive? We don't need to repeat pylint docs. 😀

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
Thanks for this PR. A small thing to check.

File: TEMPLATES/.python-lint (hunk 1 above)
Comment (Shop-kins) [reply TEMPLATES/.python-lint:11]: Will Do!

Comment (Shop-kins) [reply TEMPLATES/.python-lint:10]: have pushed a change for this :)

**Comment by ferrarimarco** [comment]
Thanks!

**Comment by ferrarimarco** [comment]
Thanks for the update.

The builds are failing because:

1. the commit message has a capital letter.
1. There's an [ad-hoc configuration file that we use for pylint when it runs against the super-linter codebase](https://github.com/super-linter/super-linter/blob/main/.github/linters/.python-lint) that we should either update, or remove.

Can you please fix the commit message?

Meanwhile, I'll check if we need that second configuration file or not.

File: TEMPLATES/.python-lint [hunk 2]
```diff
@@ -2,3 +2,9 @@
```
Comment (ferrarimarco) [inline TEMPLATES/.python-lint:2]: We don't need the super-linter specific configuration file anymore. Can you please delete `.github/linters/.python-lint` as part of this PR?

Thanks!

Comment (ferrarimarco) [reply TEMPLATES/.python-lint:2]: Actually, let me do that in a separate PR.

**Comment by ferrarimarco** [comment]
By rebasing on top of the latest `main`, the pylint check should now pass.

**Comment by Shop-kins** [comment]
> By rebasing on top of the latest `main`, the pylint check should now pass.

Have rebased and removed the capitals!

**Comment by ferrarimarco** [comment]
Thanks!

File: test/linters/python_pylint/python_good_2.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import pytest
```
Comment (ferrarimarco) [inline test/linters/python_pylint/python_good_2.py:1]: We only have two minor linting errors on this file:

```
  test/linters/python_pylint/python_good_2.py:3:13: C0303: Trailing whitespace (trailing-whitespace)
  test/linters/python_pylint/python_good_2.py:1:0: C0114: Missing module docstring (missing-module-docstring)
```

Can you please fix both?

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
A minor thing to fix, and we shall be good to go!

Thanks for this work :)

File: test/linters/python_pylint/python_good_2.py (hunk 3 above)
Comment (ferrarimarco) [reply test/linters/python_pylint/python_good_2.py:3]: Done

**Review by ferrarimarco — APPROVED** [review]

**Pull request merged**